code/datums/elements/decals/blood.dm

```diff
@@ -5,13 +5,15 @@
 		return ELEMENT_INCOMPATIBLE
 
 	. = ..()
+	ADD_TRAIT(target, TRAIT_BLOOD_ELEMENT, ELEMENT_SOURCE_TRAIT)
```

Comment:
Get rid of this trait and just use blood DNA

Reply:
I'm leaving the trait in to accommodate for mappers using element injection for now